parquet/src/arrow/async_reader/store.rs

```diff
@@ -23,6 +23,7 @@ use object_store::{path::Path, ObjectMeta, ObjectStore};
 use tokio::runtime::Handle;
 
 use crate::arrow::async_reader::AsyncFileReader;
+use crate::encryption::decryption::FileDecryptionProperties;
```

Comment:
```suggestion
#[cfg(feature = "encryption")]
use crate::encryption::decryption::FileDecryptionProperties;
```
should this be gated?

Reply:
Yes, I think so.